File: Test/RegexTesterTest.java
package test;

import static org.junit.Assert.*;

import static org.hamcrest.CoreMatchers.containsString;

import org.junit.Test;

import com.lucciola.haskell.RexTester;
import com.lucciola.result.CommunicationResult;

public class RegexTesterTest {

	@Test
	public void test() {
		RexTester tester = new RexTester();
		
		CommunicationResult result = tester.post("main = print \"Hello world\"", "");
			
		assertThat(result.getMessage(), containsString("Hello world"));
			
	}

}

File: src/test/RegexTesterTest.kt
package test

import org.junit.Assert.*

//import org.hamcrest.CoreMatchers.containsString

import org.junit.Test

import com.lucciola.haskell.RexTester
import com.lucciola.result.CommunicationResult

class RegexTesterTest {

    @Test
    fun test() {
        val tester = RexTester()

        val result = tester.post("main = print \"Hello world\"", "")

        //assertThat(result.message, containsString("Hello world"))

    }

}
